app/services/users.js

```diff
@@ -28,3 +29,20 @@ exports.createSession = async params => {
   logger.error('Password and mail mismatch for user:', params.mail);
   throw errors.authenticationError("The password and mail combination doesn't match");
 };
+
+exports.index = async req => {
+  logger.info('Index Users with: ', req);
+  const page = req.headers.page || 0;
+  const type =
+    (await jwt.findUserByToken(req.headers.authorization)).type === 'regular'
```

Comment:
Each time you want the user type you have to do this ? (A query in the database and decode the previous decoded token)